src/dotnet-svcutil/lib/Microsoft.Svcutil.NamedPipeMetadataImporter/Microsoft.Svcutil.NamedPipeMetadataImporter.csproj

```diff
@@ -1,14 +1,32 @@
 <Project Sdk="Microsoft.NET.Sdk">
-	<PropertyGroup>
+  <PropertyGroup>
+    <AssemblyVersionFile>$(IntermediateOutputPath)\$(TargetFramework)\$(MSBuildProjectName).$(TargetFramework).version.cs</AssemblyVersionFile>
+  </PropertyGroup>
+  <PropertyGroup>
 		<TargetFrameworks>net6.0;net462</TargetFrameworks>
     <RootNamespace>Microsoft.Tools.ServiceModel.Svcutil</RootNamespace>
     <AssemblyName>$(MSBuildProjectName)</AssemblyName>
     <LangVersion>latest</LangVersion>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 	</PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="System.ServiceModel.NetNamedPipe" Version="6.0.*" Condition="'$(TargetFramework)' == 'net6.0'">
-			<PrivateAssets>contentfiles;analyzers;build</PrivateAssets>
+      <IncludeAssets>all</IncludeAssets>
 		</PackageReference>
     <Reference Include="System.ServiceModel" Condition="'$(TargetFramework)' == 'net462'" />
 	</ItemGroup>
+  <PropertyGroup>
+    <CustomOutputFolder>$([System.IO.Path]::GetFullPath('$(ArtifactsBinDir)\'))\dotnet-svcutil-lib\$(Configuration)\netstandard2.0</CustomOutputFolder>
+    <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSource>false</IncludeSource>
+  </PropertyGroup>
+  <Target Name="CopyPackageDlls" AfterTargets="Build">
+    <ItemGroup>
+      <PublishedDlls Include="$(OutputPath)\*.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="@(PublishedDlls)" DestinationFolder="$(CustomOutputFolder)\internalAssets\net6.0" Condition="$(OutputPath.Contains('net6.0'))" />
```

Comment:
Is internalAssets necessary?  Will (CustomOutputFolder)\net6.0 work?

Reply:
It is necessary for the clarity and convenience of deployment, or else we will need to handle the copy of net60/net80 and net48 folders separately everywhere it is used. If "internalAssets" is not an appropriate name, we can consider a suitable alternative.